bin/activate.bat

```diff
@@ -4,14 +4,14 @@ for /f "delims=" %%i in ("%~dp0..\envs") do (
     set ANACONDA_ENVS=%%~fi
```

Comment:
This PR seems to fix the problem with the activation of environments identified by an absolute path.
As all other environments are searched under `%ANACONDA_ENVS%` there remain the following problems:
* Activation of an environment in one of the directories configured as `envs_dirs` in the `.condarc` file is not possible (at least not by name, just by full path).
* Activation of an environment specified by a relative path is not possible. This is very handy if you have a virtual environment in all of your working directories. There, the activation is done by a simple `activate .\venv` instead of `activate %CD%\venv` (which are three! more characters to type).

The second point is not so important, but should be addressed also, but the first point really is essential. And to solve it, this requires that the `activate.bat` script knows about the conda environment directories. In my old pull request, this had been solved by creating a special hidden command in conda (`..checkenv`, kudos to @ognenduz) which does all the environment checking and of course knows about all the envs directories and all environments in them.
This functionality should now go into `conda-env`, something like `conda env list` but with output for reuse in the `activate.bat` script.